pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/PrometheusMetricsTest.java

```diff
@@ -591,18 +595,63 @@ public void testManagedLedgerStats() throws Exception {
                 System.out.println(e.getKey() + ": " + e.getValue())
         );
 
+        Map<String, String> typeDefs = new HashMap<String, String>();
+        Map<String, String> metricNames = new HashMap<String, String>();
+
+        Pattern typePattern = Pattern.compile("^#\\s+TYPE\\s+(\\w+)\\s+(\\w+)");
+        Pattern metricNamePattern = Pattern.compile("^(\\w+)\\{.+");
+
+	    Splitter.on("\n").split(metricsStr).forEach(line -> {
```

Comment:
I think this block has intention issue.

Reply:
We don't use tabs in the code.

Reply:
Removed the tab. Can you clarify what's the intention issue? The code block is to catch any issues with duplicated definition of metrics TYPE and also ensures the metric name matches the ones defined in the type definition header.